packages/__docs__/src/Icons/index.tsx

```diff
@@ -112,24 +106,28 @@ class Icons extends Component {
     return glyphs
   }
 
-  handleSearchChange = (e) => {
-    const query = e.target.value
+  handleSearchChange = (e: SyntheticEvent) => {
+    const query = (e.target as HTMLInputElement).value
 
     clearTimeout(this.searchTimeout)
     this.searchTimeout = setTimeout(() => {
       this.setState({ query })
     }, 500)
   }
 
-  handleFormatChange = (e, o) => {
+  handleFormatChange = (o: Record<string, any>) => {
     window.location.hash = `#${o.value}`
   }
 
-  handleVariantClick = (e, name, variant, glyph) => {
+  handleVariantClick = (
+    name: string,
+    variant: string,
+    glyph: Record<string, any>
```

Comment:
`glyph` can be typed properly when you rebase this PR to the latest master commit, because it will contain your typings for it.